tools/compiletest/src/main.rs

```diff
@@ -175,17 +173,21 @@ pub fn parse_config(args: Vec<String>) -> Config {
         Some(x) => panic!("argument for --color must be auto, always, or never, but found `{}`", x),
     };
 
-    let src_base = opt_path(matches, "src-base");
+    let suite = matches.opt_str("suite").unwrap();
+    let src_base = opt_path(matches, "src-base", &["tests", suite.as_str()]);
```

Comment:
@adpaco-aws I was wondering if I should try to find the fit root folder or just assume we're in the top folder. What do you think?